app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt

```diff
@@ -40,6 +33,9 @@ import org.mozilla.fenix.ext.settings
 import org.mozilla.fenix.theme.ThemeManager
 import org.mozilla.fenix.utils.ToolbarPopupWindow
 import java.lang.ref.WeakReference
+import mozilla.components.browser.toolbar.behavior.ToolbarPosition as behaviorToolbarPosition
+import mozilla.components.browser.toolbar.behavior.ToolbarPosition.BOTTOM as bottomToolbarPosition
+import mozilla.components.browser.toolbar.behavior.ToolbarPosition.TOP as topToolbarPosition
```

Comment:
nit: Wondering about those named imports instead of `ToolbarPosition.*`? Makes it harder to read, at least in the PR, since those look like class members in code.

Reply:
I understand.
Thing is there already exists a aptly named `ToolbarPosition` enum in Fenix so I had to use this import aliases.
Searching now in the project using `import .* as` I see usages such as `..search.SearchEngine as LegacySearchEngine`. maybe I could use something like `MozacToolbarPosition`?

Reply:
Went with using `MozacToolbarPosition` as import alias.